### PR TITLE
INTLY-2127: add 'two RHSSO instances'  explanation

### PR DESCRIPTION
**Pull request by pwright**

### CODE REVIEW

File: docs/content/gs-adding-users-proc.adoc [hunk 1]
```diff
@@ -4,26 +4,43 @@ ifdef::env-github[]
...
+Red Hat manages this instance, however the customer-admin user has admin level privileges to configure this instance.
++
+See the link:https://access.redhat.com/documentation/en-us/red_hat_single_sign-on[Product Documentation for Red Hat Single Sign-On] for more information on using Red Hat Single Sign-On.
+
+[NOTE]
+====
+Red Hat recommends that you define users of the solution you are developing in the *{customer-sso-name}*. 
+For example, if you use {PRODUCT_SHORT} to develop an solution named `my-app`, define all users of `my-app` in the *{customer-sso-name}* instance only.
+
+Typically, you only use the *{customer-sso-name}* for end users, however you have the required privileges to use it for developer related activities.
```
Comment (david-martin) [inline docs/content/gs-adding-users-proc.adoc:36]: This line seems vague and might prompt questions about what is meant by `developer related activities`.
What's the thinking behind having it?

**Review by david-martin — APPROVED** [review]
1 question. Other than that, sounds reasonable

File: docs/content/gs-adding-users-proc.adoc (hunk 1 above)
Comment (pwright) [reply docs/content/gs-adding-users-proc.adoc:36]: there was a thread around using http://microcks.github.io/ 
so, while it's called 'end user sso', it is also where developers might work.

would it help to add ", for example, if a developer wanted to use http://microcks.github.io/[Microcks], that developer could create associated users in this instance"?

Comment (pwright) [reply docs/content/gs-adding-users-proc.adoc:36]: @david-martin  updated these paragraphs, and added upstream doc, could you give it another gander? thx

**Review by david-martin — APPROVED** [review]

**Pull request merged**